pkgs/applications/misc/darkman/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, fetchFromGitLab, buildGoModule, scdoc }:
+
+buildGoModule rec {
+  pname = "darkman";
+  version = "1.3.1";
+  nativeBuildInputs = [ scdoc ];
+
+  src = fetchFromGitLab {
+    owner = "WhyNotHugo";
+    repo = "darkman";
+    rev = "v${version}";
+    sha256 = "09iwc9cwwc88c6yrf6a552nbsnf1w8cnlra9axsar2p0k21v5yl1";
+  };
+
+  vendorSha256 = "09rjqw6v1jaf0mhmycw9mcay9q0y1fya2azj8216gdgkl48ics08";
+
+  postPatch = ''
+    substituteInPlace darkman.service \
+      --replace "/usr/bin/darkman" "$out/bin/darkman"
+    substituteInPlace contrib/dbus/nl.whynothugo.darkman.service \
+      --replace "/usr/bin/darkman" "$out/bin/darkman"
+    substituteInPlace contrib/dbus/org.freedesktop.impl.portal.desktop.darkman.service \
+      --replace "/usr/bin/darkman" "$out/bin/darkman"
+  '';
+
+  buildPhase = ''
+    make build
+  '';
+
+  installPhase = ''
+    make DESTDIR=$out PREFIX=/ install
+  '';
+
```

You can also add an update script if you want to get notified about new tags in the repo: (nix-update-script needs to be in the arguments on top)

```suggestion
  passthru.updateScript = nix-update-script {
    attrPath = pname;
  };
```